Makefile

```diff
@@ -219,22 +219,23 @@ show-version:
 	echo -n $(TAG)
 
 PLATFORMS ?= amd64 arm arm64 s390x
-
 EMPTY :=
 SPACE := $(EMPTY) $(EMPTY)
 COMMA := ,
+OUTPUT=
+
+release-ingress-controller: OUTPUT=--push
+release-ingress-controller: build-ingress-controller
```

Comment:
I'm not particularly up on my Makefile syntax, but does this actually extend the target, not redefine it?

Reply:
@brandond It extends the target definition. I tested it locally and the same concept is used here https://github.com/rancher/ingress-nginx/blob/hardened-nginx-1.2.x-fix/images/nginx/Makefile#L50